test/integration/patients/worklist/worklist.js

```diff
@@ -2541,7 +2541,7 @@ context('worklist page', function() {
       .should('contain', 'APatient AName');
   });
 
-  specify('flow sorting - patient field', function() {
+  specify('flow sorting alphabetical - patient field', function() {
```

Comment:
Split these off into two separate tests. One for alphabetical sorting and another for numerical.